cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -77,3 +77,24 @@ def test_service_ansible_playbook_negative():
         "description": fauxfactory.gen_alphanumeric()
     })
     assert not view.add.active
+
+
+@pytest.mark.tier(2)
+def test_service_ansible_playbook_confirm():
+    """Tests after selecting playbook additional widgets appear and are pre-populated where
+    possible.
+    """
+    view = navigate_to(AnsiblePlaybookCatalogItem("", "", {}), "Add")
```

Comment:
No need to actually instantiate `AnsiblePlaybookCatalogItem` here. `navigate_to(AnsiblePlaybookCatalogItem, "Add")` works as well.